public/style.css

```diff
@@ -19,3 +19,12 @@ form div {
   margin: 1em;
   display: inline-block;
 }
+
+ul {
+  padding: 0px;
+  list-style-type: none;
+}
+
+.mini {
```

Comment:
just remove the `.mini` class after you added the width attribute on the image tag (see above)